client/app/account/billing/billingApp.module.js

```diff
@@ -133,7 +133,7 @@ angular
         $stateProvider.state('app.account.billing.payment.mean.add', {
           url: '/add',
           templateUrl: `${BILLING_BASE_URL}mean/add/billing-mean-add.html`,
-          controller: 'Billing.controllers.Mean.Add',
+          controller: 'billingMeanAddCtrl',
```

Comment:
According to the state name `app.account.billing.payment.mean.add` why not naming your controller as `billingPaymentMeanAddCtrl`?